v8-upstream-master-riscv64.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import os
+import platform
+import shutil
+import subprocess
+import sys
+
+# TODO(kasperl@rivosinc.com): Enable ccache to speed up compilation on the bots.
+
+CWD = os.getcwd()
+PYTHON = sys.executable
+
+V8_ROOT_DIR = os.path.join(CWD, "v8-riscv")
+
+DEPOT_TOOLS_URL = "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
+DEPOT_TOOLS_DIR = os.path.join(CWD, "depot_tools")
+
+FETCH_PATH     = os.path.join(DEPOT_TOOLS_DIR, "fetch")
+GCLIENT_PATH   = os.path.join(DEPOT_TOOLS_DIR, "gclient")
+GN_PATH        = os.path.join(DEPOT_TOOLS_DIR, "gn")
+AUTONINJA_PATH = os.path.join(DEPOT_TOOLS_DIR, "autoninja")
+
+TARGET_CPU_MAPPING = { "x86_64": "x64" }
+TARGET_CPU = TARGET_CPU_MAPPING.get(platform.machine(), platform.machine())
+
+
+#############################################################################
+# Configurations.
+#############################################################################
+RISCV64_OPTDEBUG_SIM_CONFIG = f"""
+  is_component_build=false
+  is_debug=true
+  target_cpu="{TARGET_CPU}"
+  v8_enable_backtrace=true
+  v8_enable_slow_dchecks=true
+  v8_optimized_debug=true
```

Comment:
I'm pretty sure that this is the default, so we've been running optimized debug versions until now. I've added this line to make it explicit.